handlers/interactions/buttonInteractions.ts

```diff
@@ -338,9 +339,9 @@ async function interactionButtonHandler(interaction: ButtonInteraction) {
 
     team_players = removeEntry(team_players, interaction.user.toString());
 
-    for (const field of new_fields) {
-      if (field.name === "Players Joined") {
-        field.value = team_players.join("\n") || "";
+    for (let i = 0; i < new_fields.length; i += 1) {
```

Comment:
Can u add a comment why, this logic here , seems confusing to me, what actually it does